client/src/components/geneExpression/menus/genesetMenus.tsx

```diff
@@ -61,7 +61,9 @@ class GenesetMenus extends React.PureComponent<{}, State> {
     });
   };
 
-  handleColorByEntireGeneset = (): void => {
+  handleColorByEntireGeneset = (event: React.MouseEvent): void => {
+    event.preventDefault();
```

Comment:
just curious we didn't have `preventDefault()` before why we need it now?

Reply:
There's some weird scrollview reset behavior when coloring by genesets that I thought this fixed, lol